src/app/app.module.ts

```diff
@@ -49,6 +49,7 @@ import { DialogGenericErrorComponent } from './dialogs/dialog-generic-error/dial
 import { ProfileComponent } from './views/profile/profile.component';
 import { DialogApproveContractComponent } from './dialogs/dialog-approve-contract/dialog-approve-contract.component';
 import { DialogClientAccountComponent } from './dialogs/dialog-client-account/dialog-client-account.component';
+import { DialogLoanPayComponent } from './dialogs/dialog-loan-pay/dialog-loan-pay.component';
```

Comment:
Consider adding this dialog and pay dependencies on loan-detail.module because pay its only used for loan-detail. In case you want to use it cross on the app, should be on shared.module @Deivbid